srcs/plugin.video.playermb/main.py

```diff
@@ -521,9 +523,9 @@ def generate_m3u():
             img = item['images']['pc'][0]['mainUrl']
             img = 'https:' + img if img.startswith('//') else img
             data += '#EXTINF:-1 tvg-logo="%s",%s\n%s?mode=playm3u&channelid=%s\n' % (img, title, base_url, id)
-    f = xbmcvfs.File(M3UPATH + M3UFILE, 'w')
-    f.write(data.encode('utf-8'))
-    f.close()
+    openMode = 'w' if M3UOVERWRITE == 'true' else 'a'
```

Comment:
To oczywiście zadziała w obu Pythonach ale ciekawi mnie dlaczego zrezygnowałeś z `xbmcvfs.File`?

Reply:
W xbmcvfs brakuje modu append. Rozwiazanie z io.open moim zdaniem najprostsze :)

Reply:
I to jest bardzo dobra odpowiedź :-)